server/repository/src/migrations/v2_08_00/doses_field_on_stock_line.rs

```diff
@@ -0,0 +1,20 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "doses_field_on_stock_line"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+                ALTER TABLE stock_line ADD COLUMN vaccine_doses INTEGER;
```

Comment:
Should this not be null and just default to 0 for non-vaccine items like in item table? 👀